src/input_xml.F90

```diff
@@ -213,8 +213,8 @@ subroutine read_settings_xml()
     select case (trim(temp_str))
     case ('nuclide')
       grid_method = GRID_NUCLIDE
-    case ('union')
-      call fatal_error("Union energy grid is no longer supported.")
```

Comment:
Maybe we could have 'union' default to the material union grid with a warning message that the traditional union energy grid method no longer exists?